Translations-Web/lang_zh-tw.php

```diff
@@ -66,24 +66,24 @@
 $content_info_extraction = "並支援解壓縮這些格式：";
 
 // Info v2 (iOS and future macOS)
-$content_info_v2_title1 = "Store more";
-$content_info_v2_title2 = "Share with privacy";
-$content_info_v2_title3 = "A perfect Browsing experience";
-$content_info_v2_title4 = "Multitasking at its finest";
-$content_info_v2_title5 = "Always at hand";
-$content_info_v2_text1 = "With multiple Compression Formats<br />to choose the Perfect One";
-$content_info_v2_text2 = "Protect your shared files with a Password<br />and Encrypt them with AES-256";
-$content_info_v2_text3 = "Preview, Extract and Share<br />only what You Need";
-$content_info_v2_text4 = "Extract, Compress and Browse<br />without limits";
-$content_info_v2_text5 = "Extract, Compress and Browse<br />from anywhere using the Share Actions";
+$content_info_v2_title1 = "塞得更多";
+$content_info_v2_title2 = "私密共享";
+$content_info_v2_title3 = "瀏覽體驗無比絲滑";
```

Comment:
I doubt this is Mainland China Simplified Chinese usage.

Reply:
Not sure about it since I did not reference the existing translation.

What's your suggestion about these translation?

Reply:
As I found some other improvements, just sent the PR #1390 to include them all, please feel free to take a look.